src/server/protocol.ts

```diff
@@ -1,24 +1,41 @@
 import type * as ts from "./_namespaces/ts";
 import type {
+    ApplicableRefactorInfo,
     CompilerOptionsValue,
+    CompletionsTriggerCharacter,
     EndOfLineState,
     FileExtensionInfo,
     HighlightSpanKind,
     InteractiveRefactorArguments,
-    MapLike,
-    OutliningSpanKind,
     OutputFile,
-    PluginImport,
-    ProjectReference,
+    RefactorTriggerReason,
+    RenameInfoFailure,
     RenameLocation,
     ScriptElementKind,
     ScriptKind,
+    SignatureHelpTriggerReason,
+    SymbolDisplayPart,
     TextChange,
     TextInsertion,
     TodoComment,
     TodoCommentDescriptor,
     TypeAcquisition,
+    UserPreferences,
 } from "./_namespaces/ts";
+import {
+    ClassificationType,
+    CompletionTriggerKind,
+    OrganizeImportsMode,
+    SemicolonPreference,
+} from "./_namespaces/ts";

```

Comment:
These were identical copies of const enums previously duplicated in this file, which can simply be imported now. Since we have `preserveConstEnums` on, they have to be imported without `import type` and re-exported below to ensure the shape of this module doesn’t change. (I validated this manually by comparing `Object.keys(ts.server.protocol)` before and after this PR.)